app/controllers/frontend_log_controller.rb

```diff
@@ -0,0 +1,24 @@
+class FrontendLogController < ApplicationController
+
+  respond_to :json
+
+  def create
+    if user_fully_authenticated?
+      event, payload = log_params
+      analytics.track_event(event, payload)
```

Comment:
I don't know if we care much to restrict it, but would we want to create a list of allowable / expected events? Or otherwise "scope" (prefix) events separate from those in the `Analytics::` set of constants?

Reply:
Yeah, or perhaps a helper on the `Analytics` class that returns a set of permissible events for the fronted?

Reply:
Do we have a list of permissible/allowable events?

Reply:
I think we should not restrict to allowable events, because if we did, then during a deploy when the frontend may know about newer events and a request goes to an old backend server, we'd reject the event

Reply:
Hmm, I'm thinking that as long as we don't error on rejected events that shouldn't be a problem 🤔. If we change the events we're going to be dropping new events anyway because old servers won't be sending them.

I think the list of permissible events is advantageous in that is makes it more difficult to fill our logs w/ junk.

Reply:
aduth showed me and we're now limiting it to the two we care about.